### DISCUSSION TITLE
Performance a bit problematic. Is there an active adr-tools fork for madr?

### DISCUSSION THREAD

**Discussion by chipbite** [thomvaill/log4brains#53] [Feedback]
Hi there!

I wanted to try madr commandline tool support. Found that the fork of adr-tools has been archived.
Proceeded to install this instead.
It took some time and added **558** packages! Maybe I need to update my node install or something?

```shell
npm install log4brains -g
[...]
+ log4brains@1.0.0-beta.11
added 558 packages from 504 contributors in 138.945s
```

Then, I wanted to check the help message, so I ran `log4brains --help`. It took it's sweet time!

```shell
$ time log4brains --help                                  Usage: log4brains [options] [command]

Log4brains CLI to preview and build your architecture knowledge base.
You can also manage your ADRs from here (see `log4brains adr --help`).

All the commands should be run from your project's root folder.

Add the `--help` option to any command to see its detailed documentation.

Options:
  -V, --version            output the version number
  -h, --help               display help for command

Commands:
  init [options] [path]    Configures Log4brains for your project
  adr                      Group of commands to manage your ADRs...
  preview [options] [adr]  Start Log4brains locally to preview your changes
  build [options]          Build Log4brains as a deployable static website
  help [command]           display help for command

real    0m12.231s
user    0m0.105s
sys     0m0.618s
```

12+ seconds to get to see 17 lines of help. 
Haha, computing in 2020s - slower than ever? 😊🤔😉 Ok, bad joke.

But I was wondering - is there any lightweight option that one could use, maybe until the beta is done? 
Like adr-tools for madr? A nice lightweight shell script to do these simple things are maybe not so bad.

`<jokes>` I might just *gasp* use cp & mkdir instead... ;-) 
Now trying `log4brains init`, hope it finishes thinking today... `</jokes>`

Really cool project though, I think the ideas are awesome! Thanks alot!

BR! /marcus

**Comment by Philzen** [comment]
Looks like @thomvaill has already identified a potential bottleneck, see #19 (although there has not been any updates progress since Jan 2021)..

**Reply by thomvaill** [reply]
#19 is fixed now :-)
